<launch>
    
    <node pkg="magabot_joy" type="teleop.py" name="robot_teleop" output="screen">
        <param name="max_vel" value="1.0" />
        <param name="min_vel" value="0.1" />
        <param name="max_ang" value="1.57" />
        <param name="min_ang" value="0.5" />
        <param name="def_lvel" value="0.3" />
        <param name="def_avel" value="0.5" />
        <param name="secStop" value="5.0" />
        <param name="secRestart" value="30.0" />
    </node>

</launch>
